style.scss

```diff
@@ -0,0 +1,924 @@
+    display: block;
+    transition: 0.3s;
+}
+
+.overlay4 a:hover, .overlay4 a:focus {
+    color: #f1f1f1;
+}
+
+.overlay4 .closebtn {
+    background-color: #ce1126;
+    position: absolute;
+    top: 20px;
+    right: 45px;
+    font-size: 60px;
+}
+
+@media screen and (max-height: 450px) {
+    .overlay4 a {font-size: 20px}
+    .overlay4 .closebtn {
+        font-size: 40px;
+        top: 15px;
+        right: 35px;
+    }
+}
+/* =============End of Card 4 open pages===========*/
+
+
+/* =============Start of CSS for 5 open pages===========*/
+
+.overlay5 {
+  /*background-color: black;*/
+    background-color: #9b59b6;
+}
+
+.overlay5 a {
+    padding: 8px;
+    text-decoration: none;
+    font-size: 36px;
+    color: lightgray;
+    display: block;
+    transition: 0.3s;
+}
+
+.overlay5 a:hover, .overlay5 a:focus {
+    color: #f1f1f1;
+}
+
+.overlay5 .closebtn {
+    background-color: #9b59b6;
+    position: absolute;
+    top: 20px;
+    right: 45px;
+    font-size: 60px;
+}
+
+@media screen and (max-height: 450px) {
+    .overlay5 a {font-size: 20px}
+    .overlay5 .closebtn {
+        font-size: 40px;
+        top: 15px;
+        right: 35px;
+    }
+}
+/* =============End of Card 5 open pages===========*/
+
+
+/* =============Start of CSS for 6 open pages===========*/
+.overlay6 {
+    /*background-color: black;*/
+    background-color: #3498db;
+}
+
+.overlay6 a {
+    padding: 8px;
+    text-decoration: none;
+    font-size: 36px;
+    color: #818181;
+    display: block;
+    transition: 0.3s;
+}
+
+.overlay6 a:hover, .overlay6 a:focus {
+    color: #f1f1f1;
+}
+
+.overlay6 .closebtn {
+    background-color: #3498db;
+    position: absolute;
+    top: 20px;
+    right: 45px;
+    font-size: 60px;
+}
+
+@media screen and (max-height: 450px) {
+    .overlay6 a {font-size: 20px}
+    .overlay6 .closebtn {
+        font-size: 40px;
+        top: 15px;
+        right: 35px;
+    }
+}
+/* =============End of CCard 6 open pages===========*/
+
+
+/*!* =============Start of CSS for 7 open pages===========*!*/
+
+.overlay7 {
+    /*background-color: black;*/
+    background-color: #1abc9c;
+}
+
+.overlay7 a {
+    padding: 8px;
+    text-decoration: none;
+    font-size: 36px;
+    color: #818181;
+    display: block;
+    transition: 0.3s;
+}
+
+.overlay7 a:hover, .overlay7 a:focus {
+    color: #f1f1f1;
+}
+
+.overlay7 .closebtn {
+    background-color: #1abc9c;
+    position: absolute;
+    top: 20px;
+    right: 45px;
+    font-size: 60px;
+}
+
+@media screen and (max-height: 450px) {
+    .overlay7 a {font-size: 20px}
+    .overlay7 .closebtn {
+        font-size: 40px;
+        top: 15px;
+        right: 35px;
+    }
+}
+/* =============End of Card 7 open pages===========*/
+
+.responsive{
+    width: 100%;
+    height: auto;
+    max-width: 550px;
+    padding: 20px;
+}
+
+
+/*==================================================================*/
+/*=====Images Overlay within content info cards starts here ========*/
+.image_card1{
```

Comment:
Same thing about re-using the layout rules and customizing the image-specific stuff applies here.